__integration_tests__/IntegrationUtil.js

```diff
@@ -0,0 +1,8 @@
+const spawnSync = require('child_process').spawnSync;
```

Comment:
Please add the FB copyright header and 'use strict';. Unfortunately we don't have an eslint rule we could open source for that and it is written in PHP in www I believe.